package org.folio.rest.workflow.exception;

public class SecureOperationException extends RuntimeException {

  public SecureOperationException(String message, Throwable cause) {
    super(message, cause);
  }

}
